src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserControllerIntTest.kt

```diff
@@ -0,0 +1,52 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.resource
+
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.nomisuserrolesapi.integration.IntegrationTestBase
+
+class UserControllerIntTest : IntegrationTestBase() {
+
+  @Test
+  fun `access forbidden when no authority`() {
+
+    webTestClient.get().uri("/users/testuser1")
+      .exchange()
+      .expectStatus().isUnauthorized
+  }
+
+  @Test
+  fun `access forbidden when no role`() {
+
+    webTestClient.get().uri("/users/testuser1")
+      .headers(setAuthorisation(roles = listOf()))
+      .exchange()
+      .expectStatus().isForbidden
+  }
+
+  @Test
+  fun `get user not found`() {
+
+    webTestClient.get().uri("/users/dummy")
+      .headers(setAuthorisation(roles = listOf("ROLE_USER_ADMIN")))
+      .exchange()
+      .expectStatus().isNotFound
+  }
+
+  @Test
+  fun `get user`() {
+
+    webTestClient.get().uri("/users/testuser1")
+      .headers(setAuthorisation(roles = listOf("ROLE_USER_ADMIN")))
+      .exchange()
+      .expectStatus().isOk
+      .expectBody().json(
```

Comment:
Can we have a data helper to add "John Smith" to the database rather than allow on seed data?